release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2086,6 +2086,15 @@ As a temporary workaround, the image registry should not be configured as privat
 
 [id="ocp-telco-core-4-17-known-issues_{context}"]
 
+* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, isolated CPUs are listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS`. However, changing the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing does not increase, as the reserved CPUs remain in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
++
+[NOTE]
+====
+The `IRQBALANCE_BANNED_CPULIST` variable and the `IRQBALANCE_BANNED_CPUS` variable are stored in `/etc/sysconfig/irqbalance`.
+====
+
+* Each `NodeGroup` must only match one `MachineConfigPool`. In some cases, the NUMA Resources Operator can allow a configuration where a `NodeGroup` matches more than one `MachineConfigPool`. This issue could lead to unexpected behavior in resource management. (link:https://issues.redhat.com/browse/TELCODOCS-2051[*TELCODOCS-2051*])
```

Comment:
`NodeGroup` -> node group || `NodeGroup` resource

Or something like that. Same for other API objects. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting to fix.

Reply:
Updated as per the suggestion.